### PR TITLE
New Changes been made in contest, config and TestLogin files

### PR DESCRIPTION
**Pull request by Mehwish93**
# Conftest: 
- Email and Password methods been created in conftest file.
- Added email and password content in Pytest_adoption method

# Config:
- Email is stored in an email variable in config file.
- Password is stored in an email variable in config file.

# TestLogin:
Used selenium library for login method

# Note:

- I didn't add "require = True" in Pytest_adoption method because it was giving me an error.
- I made changes in conftest branch then push it packages branch first.
- Now sending you for approval for pushing it in master branch.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+url = "http://automationpractice.com/index.php"
+email = "mehwishpeter.est@gmail.com"
+password = "Testing1234"
```
Comment (abhilashcae) [inline config.py:4]: Please remove email and password as we're taking them rom command line args.

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import config
+from tests.test_login import TestLogIn
+from utilities.driver_factory import Driver
+
+d = Driver()  # Created an instance
+loginep = TestLogIn()
```
Comment (abhilashcae) [inline tests/conftest.py:8]: I think this is an unused code and can be deleted.

File: tests/test_login.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from selenium import webdriver
+import time
+
+import config
+
+
+class TestLogIn:
```
Comment (abhilashcae) [inline tests/test_login.py:7]: Login (lowercase "i") 😃 as it's a single word

**Comment by Mehwish93** [comment]
All changes are made that had been asked.

**Pull request closed without merging**